### PR TITLE
removed comment and discussion headings

### PR DESCRIPTION
**Pull request by garrett**
Addressing https://github.com/oVirt/ovirt-site/pull/476#issuecomment-248116763 — but this might not leave just empty sections. Review and additional changes required before merging.

### CODE REVIEW

File: source/develop/release-management/features/infra/taskmanagercanceltask.html.md [hunk 1]
```diff
@@ -110,7 +110,6 @@ The cancellation of the task is depend on VDSM API, on internal Async task mange
 
 ### Documentation / External references
 
-### Comments and Discussion
```
Comment (rollandf) [inline source/develop/release-management/features/infra/taskmanagercanceltask.html.md]: This one should stay.

File: source/develop/release-management/features/network/device-custom-properties.html.md [hunk 2]
```diff
@@ -151,7 +151,6 @@ As this is a 3.3 feature, all 3.2 (and down) cluster related entities should not
 *   [Features/Quantum_Integration](Features/Quantum_Integration)
 *   Almost any interesting hook for [hotplug disk](https://bugzilla.redhat.com/show_bug.cgi?id=908656) is going to require per-disk triggering proprty
 
-## Comments and Discussion
```
Comment (rollandf) [inline source/develop/release-management/features/network/device-custom-properties.html.md]: This one should stay.

File: source/develop/release-management/features/network/host-network-qos.html.md [hunk 3]
```diff
@@ -62,7 +62,6 @@ The following steps should be tested:
 11. Functional test - initiate usage of the network to which QoS had been applied, monitor the traffic on it and verify that it is capped according to the QoS limitations.
 12. Move the host to a different cluster, whose compatibility version is smaller than 3.4; verify that it is no longer possible to override the QoS configuration on an interface via the Setup Networks dialog.
 
-### Comments and Discussion
```
Comment (rollandf) [inline source/develop/release-management/features/network/host-network-qos.html.md]: This one should stay.

File: source/develop/release-management/features/network/migration-network.html.md [hunk 4]
```diff
@@ -194,7 +194,6 @@ Note that the migration protocol requires Vdms-Vdsm and libvirt-libvirt communic
 
 *   Yuval M asking to choose a network for migration data: <http://lists.ovirt.org/pipermail/users/2013-January/011301.html>
 
-## Comments and Discussion
```
Comment (rollandf) [inline source/develop/release-management/features/network/migration-network.html.md]: This one should stay.

File: source/develop/release-management/features/network/network-custom-properties.html.md [hunk 5]
```diff
@@ -214,7 +214,6 @@ This would execute an ethtool process for each slave.
 *   <https://bugzilla.redhat.com/show_bug.cgi?id=1080984>
 *   <https://bugzilla.redhat.com/show_bug.cgi?id=1080987>
 
-## Comments and Discussion
```
Comment (rollandf) [inline source/develop/release-management/features/network/network-custom-properties.html.md]: This one should stay.

File: source/develop/release-management/features/virt/spicerelatedfeatures.html.md [hunk 6]
```diff
@@ -162,7 +162,6 @@ Here are the GUI mock-ups for the New/Edit VM/Template/VM-Pool dialogs:
     -   It is relevant only in VMs/Templates of cluster level 3.1 and above. For cluster level 3.0, if USB is enabled, "Legacy" will be used automatically.
     -   API-Engine mapping: If "enabled" is "true" then "Legacy" will be mapped to EnabledLegacy, and "Native" will be mapped to EnabledNative. If "enabled" is "false" then it will be mapped to Disabled (regardless of the value in the "type" element).
 
-## Comments and Discussion
```
Comment (rollandf) [inline source/develop/release-management/features/virt/spicerelatedfeatures.html.md]: This one should stay.

File: source/develop/release-management/features/storage/move-disk-two-phase-commit.html.md [hunk 7]
```diff
@@ -116,7 +116,6 @@ End with failure - Check if the image exists in the source storage, if so will c
 
 undeleted image from the storage should be reflected in the engine with ILLEGAL status
 
-## Comments and Discussion
```
Comment (rollandf) [inline source/develop/release-management/features/storage/move-disk-two-phase-commit.html.md]: This one should stay.

**Comment by rollandf** [comment]
:+1:  Looks good

**Comment by garrett** [comment]
Thanks, @rollandf!

I've rebased on master (and dropped the accidental data/events upgrade in the process) and did a force-push to this branch. After CI finishes, I'll squash-merge it to be one commit.

**Pull request merged**